Tests/WitsmlExplorer.IntegrationTests/Resources/attachment.xml

```diff
@@ -0,0 +1,21 @@
+<attachments xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns="http://www.witsml.org/schemas/1series" xsi:schemaLocation="http://www.witsml.org/schemas/1series  ../xsd_schemas/obj_attachment.xsd" version="1.4.1.1">
+  <attachment uidWell="8c77de13-4fad-4b2e-ba3d-7e6b0e35a394" uidWellbore="ae75db48-5cef-4bd1-9ddf-6035b0d2cd49" uid="attachmentUid">
+    <nameWell>my well</nameWell>
+    <nameWellbore>my wellbore</nameWellbore>
+    <name>1</name>
+    <objectReference object="rig" uidRef="Rig_1">Rig 1</objectReference>
+    <subObjectReference object="3" uidRef="4">subObjRef</subObjectReference>
+    <md uom="m">123</md>
+    <mdBit uom="m">456</mdBit>
+    <typeMessage>type</typeMessage>
```

Comment:
Remove typeMessage as it does not belong in an attachment.

Reply:
done